api/envoy/config/core/v3/protocol.proto

```diff
@@ -301,9 +301,12 @@ message Http2ProtocolOptions {
   // of PRIORITY frames received over the lifetime of connection exceeds the value calculated
   // using this formula::
   //
-  //     max_inbound_priority_frames_per_stream * (1 + inbound_streams)
+  //   max_inbound_priority_frames_per_stream * (1 + opened_streams)
   //
-  // the connection is terminated. The ``http2.inbound_priority_frames_flood`` stat tracks
+  // the connection is terminated. For downstream connections the `opened_streams` is incremented when
+  // Envoy receives complete response headers from the upstream server. For upstream connection the
```

Comment:
@yanavlasov Is this comment right? I'm having a hard time seeing where that's happening. It looks to me like it's happening when receiving the first header frame.